src/dstack/plugins/builtin/rest_plugin/_plugin.py

```diff
@@ -0,0 +1,118 @@
+import json
+import os
+
+import requests
+from pydantic import ValidationError
+
+from dstack._internal.core.errors import ServerClientError
+from dstack._internal.core.models.fleets import FleetSpec
+from dstack._internal.core.models.gateways import GatewaySpec
+from dstack._internal.core.models.volumes import VolumeSpec
+from dstack.plugins import ApplyPolicy, ApplySpec, Plugin, RunSpec, get_plugin_logger
+from dstack.plugins.builtin.rest_plugin import (
+    FleetSpecRequest,
+    FleetSpecResponse,
+    GatewaySpecRequest,
+    GatewaySpecResponse,
+    RunSpecRequest,
+    RunSpecResponse,
+    SpecApplyRequest,
+    SpecApplyResponse,
+    VolumeSpecRequest,
+    VolumeSpecResponse,
+)
+
+logger = get_plugin_logger(__name__)
+
+PLUGIN_SERVICE_URI_ENV_VAR_NAME = "DSTACK_PLUGIN_SERVICE_URI"
+PLUGIN_REQUEST_TIMEOUT = 8  # in seconds
```

Comment:
```suggestion
PLUGIN_REQUEST_TIMEOUT_SEC = 8  # in seconds
```

Not sure if this would need to be updated elsewhere